Sommersalat_Tomaten_Wassermelone.md

```diff
@@ -0,0 +1,39 @@
+# Sommersalat von Tomaten und Wassermelone
+
+*für 2 Personen*
+
+## Zutaten
+
+* 4 - 5 Rispentomaten mittelgross
+* 1/2 Wassermelone
+* 100g Fetakäse
+* 1 TL schwarzer Pfeffer gemahlen
+* 1 TL Koriandersamen gemahlen
+* 1/2 TL Kreuzkümmel gemahlen
+* 1/2 TL Kurkuma gemahlen
+* 1 EL Kokosnussöl
+* 2 EL mildes natives Olivenöl
+* 1 Limette
+* Meersalz
+* glatte Petersilie zum dekorieren
+
+## Zubereitung
+
+Koriander, Pfeffer, Kreuzkümmel und das Kurkumapulver zusammen mit den Ölen in einen kleinen Topf geben und etwa 2 Minuten köcheln lassen, bis die Gewürze duften.
+Das Öl durch ein Sieb abseihen und abkühlen lassen.
+Danach eine Prise Meersalz und den Saft einer halben Limette einrühren.
+
+Die Tomaten waschen und in Würfel schneiden.
+Die halbe Melone ebenfalls in Würfel schneiden.
+Tomate und Melone in einer Schüssel mischen.
+Den Fetakäse ebenfalls würfeln und in die Schüssel geben.
+Das Gewürzöl über die Zutaten gießen und alles vorsichtig vermengen.
+Mindestens 15 Minuten im Kühlschrank durchziehen lassen.
+Bei Bedarf kann auch noch Limettensaft nachgegeben werden.
```

Comment:
Vielen Dank für Dein Rezept. 
Aber bitte unterteile die Zubereitung noch in kleinere einzeln nummerierte Schritte. 
Und es wäre toll wenn du noch das Foto einbinden könntest. Denn das sieht gut aus.

Reply:
Hallo, das habe ich gern angepasst. Allerdings bereitet mir das Bild Schwierigkeiten... in der Vorschau ist es nur als Symbol vorhanden, wird aber nicht angezeigt, obwohl ich es inzwischen schon verkleinert habe..
Ich verwende den Befehl:
<img src="URL"/>
Hat jemand eine Idee?
Gruß Jörg

Reply:
Hallo wo16c,

Ich hab mir das mal ein bisschen angeguckt. Wenn ich das richtig sehe, hast du deine Markdown-Datei direkt in dem Repository und nicht in dem Unterordner Beilagen/Salate/ drin. Das solltest du als erstes ändern.
Dazu ziehst du die Datei lokal wie gewohnt im Datei-Explorer oder im Finder in den gewünschten Unterordner (lets_git_kochbuch/Beilagen/Salate/). Dorthin solltest du dann auch das Image schieben.

Bevor du das ganze committest, kannst du dein Bild wie folgt in deine Sommersalat_Tomaten_Wassermelone.md einbinden:
<img src="Salat_Tomate_Wassermelone.jpg" style="max-width:100%">
(So kannst du das immer dann machen, wenn die Datei die du einbinden willst, im selben Ordner liegt)

Dann musst du beides committen und pushen.
Meine Erfahrung damit:
Als ich meine Markdown-Datei in einem Unterordner committen wollte, musste ich zurück in das Hauptrepository wechseln (lets_git_kochbuch) und von dort aus den Commit durchführen und beide Unterordner (in deinem Fall Beilagen/Salate/) mit committen. Dadurch sollte der Inhalt von Salate/ (also deine .md und dein .jpg) mit committet werden.

Ich hoffe, dass ist verständlich formuliert und funktioniert. Viel Glück! ^^